docs/modules/reference/pages/glossary.adoc

```diff
@@ -73,16 +98,25 @@ Use to generate requisition XML files for creating/updating/removing nodes for m
 Provisioning:: The process of getting your devices, applications, and services into monitoring.
 (See xref:operation:provisioning/introduction.adoc#discovery-auto[Provisioning].)
 
-Remote Method Invocation (RMI):: Java API that lets one Java Virtual Machine (JVM) running object to invoke methods on an object running in another JVM. RMI integration lets you access Horizon/Meridian through jconsole or remote manage them.
+Remote Method Invocation (RMI):: Java API that lets one Java Virtual Machine (JVM) running object to invoke methods on an object running in another JVM. 
+RMI integration lets you access Horizon/Meridian through jconsole or remote manage them.
 (See xref:operation:admin/rmi.adoc#rmi[Enabling RMI].)
 
+Requisitions:: Sets of nodes to import into {page-component-title} for monitoring and management.
+Requisitions can be iteratively built and imported at a later date.
+(See xref:operation:provisioning/getting-started.adoc#requisition-create[Create a Requisition].)
+
 Round Robin Database (RRD):: Special type of database designed for collection, visualization, and analysis of time-series data.
 Data is maintained as a fixed-size circular-buffer that overwrites the oldest data with new data.
 
 Sentinel:: Karaf container that provides scalability for data processing, including flows, SNMP traps, syslog messages, and streaming telemetry in OpenNMS.
 It also supports thresholding for streaming telemetry if you are using OpenNMS Newts.
 (See xref:deployment:sentinel/introduction.adoc#sentinel[Sentinel].)
```

Comment:
Needs some updates to align with the Sentinel Deployment docs that state only flows are supported at this time.
```suggestion
Sentinel:: Karaf container that provides scalability for data processing, including flows, SNMP traps, syslog messages, and streaming telemetry in OpenNMS.
It also supports thresholding for streaming telemetry if you are using OpenNMS Newts.
(See xref:deployment:sentinel/introduction.adoc[Sentinel].)
```

Reply:
Done; will be included in my next commit.